lib/ddtrace/contrib/sidekiq/configuration/settings.rb

```diff
@@ -17,6 +17,11 @@ class Settings < Contrib::Configuration::Settings
             o.lazy
           end
 
+          option :log_job_arguments do |o|
+            o.default { env_to_bool(Ext::ENV_LOG_JOB_ARGUMENTS, false) }
```

Comment:
I think this is a good idea defaulting to `false`, since some args might be too sensitive to tag. :+1: